modules/social_features/social_group/src/Plugin/Field/FieldWidget/SocialGroupSelectorWidget.php

```diff
@@ -262,7 +262,7 @@ public static function validateGroupSelection(array $form, FormStateInterface $f
 
       $ajax_response->addCommand(new InvokeCommand('#edit-field-content-visibility-' . $visibility, 'change'));
     }
-    $text = t('Changing the group may have impact on the <strong>visibility settings</strong>.');
+    $text = t('Changing the group may have impact on the <strong>visibility settings</strong> and <strong>access</strong> for <strong>author/co-authors</strong> of the current page.');
```

Comment:
Would be great if we can remove the specific "of the current page" as this will be shown on events, nodes etc.

Just

`t('Changing the group may have impact on the <strong>visibility settings</strong> and may cause <strong>author/co-authors</strong> to lose access.)`

Reply:
done)